README.md

```diff
@@ -20,7 +20,6 @@ This assumes that you're running Linux or MacOS and have installed [Go 1.15+](ht
 If you already have a previous version of Decentr installed:
 ```
 rm -rf ~/.decentrd
```

Comment:
.decentrd => .decentr ?

Reply:
No, d is being be added in cosmoscmd package